docs/source/release/v6.2.0/mantidworkbench.rst

```diff
@@ -8,52 +8,57 @@ Mantid Workbench Changes
 New and Improved
 ----------------
```

Comment:
Could we split new from improved? There are 2 new widgets, it's better to make them stand out.